english/LoveLive/lovelive.md

```diff
@@ -0,0 +1,22 @@
+Love Live! School Idol Project (ラブライブ! School idol project) is a Japanese multimedia project
+ co-developed by ASCII Media Works' Dengeki G's Magazine, music label Lantis, and animation
+ studio Sunrise. The project revolves around a group of fictional school girls who become
+ idols in order to save their school from shutting down. It launched in the August 2010 issue
+ of Dengeki G's Magazine, and went on to produce music CDs, anime music videos, two manga
+ adaptations, and video games.
+
+A 13-episode [anime television series](http://myanimelist.net/anime/15051/Love_Live_School_Idol_Project)
+ produced by Sunrise and directed by Takahiko Kyōgoku aired in Japan between January and March 2013, with a
+ [second season](http://myanimelist.net/anime/19111/Love_Live_School_Idol_Project_2nd_Season) airing between
+ April and June 2014. An [animated film](http://myanimelist.net/anime/24997/Love_Live_The_School_Idol_Movie)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.